packages/kms-web3/package.json

```
@@ -10,12 +10,10 @@
     "extract-api": "node ../cli/bin/veramo.js dev extract-api"
   },
   "dependencies": {
-    "@ethersproject/providers": "^5.7.0",
-    "@ethersproject/strings": "^5.7.0",
-    "@ethersproject/transactions": "^5.7.0",
     "@veramo/core-types": "workspace:^",
     "@veramo/key-manager": "workspace:^",
-    "debug": "^4.3.3"
+    "debug": "^4.3.3",
+    "ethers": "6.7.1"
```

Please accept wider version ranges for dependencies
```suggestion
    "ethers": "^6.7.1"
```